generate_nb_vocab_file.py

```diff
@@ -0,0 +1,54 @@
+import inspect
+from pathlib import Path
+
+import pydantic
+
+from bagel import mappings, models
+
+EXCLUDE_CLASSES = ["Bagel", "BaseModel"]
+PREAMBLE = "\n".join(
+    [
+        "@prefix nb: <http://neurobagel.org/vocab/>.",
+        "@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.",
+    ]
+)
+OUTPUT_PATH = Path.cwd() / "nb_vocab.ttl"
+
+
+def generate_vocab_string_from_models():
+    all_classes = {
+        name: obj
+        for name, obj in inspect.getmembers(models)
+        if inspect.isclass(obj)
+        and issubclass(obj, pydantic.BaseModel)
+        and name not in EXCLUDE_CLASSES
+    }
+    nb_vocab = ""
+
+    for count, (name, obj) in enumerate(sorted(all_classes.items())):
+        class_entry = f"{mappings.NB.pf}:{name}\n" "    a rdfs:Class"
```

Comment:
```suggestion
        class_entry = f"{mappings.NB.pf}:{name}\n    a rdfs:Class"
```
Was the string end here intentional? Why is the `a rdfs:Class` on a newline?